app/services/duplicate_application.rb

```diff
@@ -50,16 +50,25 @@ def duplicate
       new_application_form.update!(degrees_completed: false)
     end
 
-    original_application_form.application_references.includes([:reference_tokens]).where(feedback_status: %w[feedback_provided not_requested_yet cancelled_at_end_of_cycle feedback_requested]).reject(&:feedback_overdue?).each do |w|
+    original_references = original_application_form.application_references
+      .includes([:reference_tokens])
+      .where(feedback_status: %w[feedback_provided not_requested_yet cancelled_at_end_of_cycle feedback_requested])
+      .reject(&:feedback_overdue?)
+
+    original_references.each do |original_reference|
       new_application_form.application_references.create!(
-        w.attributes.except(*IGNORED_CHILD_ATTRIBUTES).merge!(duplicate: true),
+        original_reference.attributes.except(*IGNORED_CHILD_ATTRIBUTES).merge!(duplicate: true),
       )
 
+      if FeatureFlag.active?(:new_references_flow)
+        awaiting_response_references = new_application_form.application_references.select(&:feedback_requested?)
```

Comment:
i think rails enum trickery generates class methods such that you can refer to `new_application_form.application_references.feedback_requested`